frontend/pkg/frontend/ocm.go

```diff
@@ -639,7 +638,7 @@ func (f *Frontend) BuildCSExternalAuth(ctx context.Context, externalAuth *api.HC
 				UserName(arohcpv1alpha1.NewUsernameClaim().
 					Claim(externalAuth.Properties.Claim.Mappings.Username.Claim).
 					Prefix(externalAuth.Properties.Claim.Mappings.Username.Prefix).
-					PrefixPolicy(externalAuth.Properties.Claim.Mappings.Username.PrefixPolicy),
+					PrefixPolicy(string(externalAuth.Properties.Claim.Mappings.Username.PrefixPolicy)),
```

Comment:
We should skip setting these if the string is empty.  Same for `URL` and `CA` above (and maybe elsewhere).  We'll have to break up the code a bit so it's not just one giant chain of builder calls.

Reply:
Why? If a user sets it as an empty string, we should honor that and send an empty string to CS right? What we do need to do though is not set it if the value is 'nil' i.e. make it a pointer in the first place